connectors/class-connector-acf.php

```diff
@@ -479,10 +678,31 @@ public function log_override( $data ) {
 			return $data;
 		}
 
-		if ( 'posts' === $data['connector'] && 'acf' === $data['context'] ) {
-			$data['context']               = 'field_groups';
-			$data['connector']             = $this->name;
-			$data['args']['singular_name'] = esc_html__( 'field group', 'stream' );
+		$is_acf_context = 'acf' === $data['context']
```

Comment:
@kidunot89 how about this: 

```php
$is_acf_context = in_array( $data['context'], array( 'acf', 'acf-field-group', 'acf-field' ), true );
```

Reply:
:man_facepalming: Can't believe I didn't think of that :joy: